docs/user-guide/content-sources.rst

```diff
@@ -98,6 +103,7 @@ The pulp-admin client can be use to list all defined content sources as follows:
   Base URL:       http://content-world/content/
   Enabled:        1
   Expires:        3d
+  Headers:        X-RHUI-ID=1234 \ X-Csrf-Token=31415926535897
```

Comment:
Suggest you remove the '\'  here.  Also, I think we want to encourage each header on a separate line for clarity and consistency with previous example.